### ISSUE TITLE
Installing with Bower throws an error

### ISSUE THREAD

**Issue by brentajones** [MinnPost/legislature-tracker#42]
Running:
     `bower install legislature-tracker`
Returns:

```
    ...
     bower Ractive-events-tap#~0.1.0        ENOTFOUND Package Ractive-events-tap not found
```

**Comment by zzolo** [comment]
Yeah, that has since been updated; the repo moved and Bower package changed.  You can change that line in `bower.json` to the following and I'll leave this open until it's updated in the code:

`"ractive-events-tap": "~0.1.0"`
